MoreLinq/Aggregate.g.cs

```diff
@@ -15,6 +15,16 @@
 // limitations under the License.
 #endregion
 
+#nullable enable // required for auto-generated sources (see below why)
```

Comment:
Would argue these should be done in a separate PR, either before or after this one.

Reply:
Agree, but for now these were needed to make the CI build for this PR pass. I'll be moving these changes out.

Reply:
Tracked as bug #981.